src/hook.c

```diff
@@ -359,14 +366,17 @@ static VOID Unfreeze(PFROZEN_THREADS pThreads)
 {
     if (pThreads->pItems != NULL)
     {
-        UINT i;
-        for (i = 0; i < pThreads->size; ++i)
+        for (UINT i = 0; i < pThreads->size; ++i)
         {
-            HANDLE hThread = OpenThread(THREAD_ACCESS, FALSE, pThreads->pItems[i]);
-            if (hThread != NULL)
+            DWORD threadId = pThreads->pItems[i];
+            if(threadId > 0)
```

Comment:
Curious why this change is required?

Reply:
The space - for formatting.
The operator - maybe a matter of taste, doesn't matter in this case since `threadId` is unsigned, but I think that `!=` is clearer since that's what you're actually after. With `>`, a reader might wonder about the meaning of negative thread ids, only to realize that the value is unsigned.

Reply:
Good point, thanks for the insight! :)